### PR TITLE
move podresources endpoint to beta

### PR DESCRIPTION
**Pull request by dashpole**
/milestone 1.15
Related Enhancement: https://github.com/kubernetes/enhancements/issues/606
The feature is moving to beta in 1.15

### CODE REVIEW

**Comment by sftim** [comment]
Simple PR, seems easy to review.
/lgtm

File: content/en/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins.md [hunk 1]
```diff
@@ -164,7 +164,7 @@ DaemonSet, `/var/lib/kubelet/pod-resources` must be mounted as a
 in the plugin's
 [PodSpec](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#podspec-v1-core).
 
-Support for the "PodResources service" is still in alpha.
+Support for the "PodResources service" is in beta, and is enabled by default.
```
Comment (xiangpengzhao) [inline content/en/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins.md]: This change LGTM. However, I see this `Kubernetes device plugin support is still in alpha.` in section `Device plugin deployment`. AFAIK, Kubernetes device plugin support is already in beta from v1.14. See: https://kubernetes.io/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins/

Comment (dashpole) [reply content/en/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins.md]: It sounds like we need to make that change separately, and cherry-pick it to the 1.14 release docs.

Comment (sftim) [reply content/en/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins.md]: It'd be more usual for website docs to make the change in `master` and not worry about putting the equivalent change into the v1.15 branch

Right now the most recent Kubernetes release is 1.14, so `master` is documentation for cluster operators running 1.14

Comment (xiangpengzhao) [reply content/en/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins.md]: The sentence there in [master branch](https://github.com/kubernetes/website/blob/master/content/en/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins.md) is `Kubernetes device plugin support is in beta.`
The sentence there in [1.14 release branch](https://github.com/kubernetes/website/blob/release-1.14/content/en/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins.md) is `Kubernetes device plugin support is still in alpha.`
But the sentence there in the published [docs website](https://kubernetes.io/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins/) (which means to show 1.14 release contents) is `Kubernetes device plugin support is in beta.`
And the sentence there in the [forked branch](https://github.com/kubernetes/website/blob/ce47ddd5f7fbd8db1e84513ad84e4a8f73496c66/content/en/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins.md) of this PR is `Kubernetes device plugin support is still in alpha.`

I'm confused... @tengqm any thoughts?

Comment (kbhawkey) [reply content/en/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins.md]: OK, I would check that the latest changes from master are up-to-date in the dev-1.15 branch. I think the change in question happened a few days ago? The dev-1.15 branch should contain the master branch changes plus the new 1.15 content.
/cc @makoscafee

Comment (makoscafee) [reply content/en/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins.md]: Hi, @kbhawkey will work to update dev-1.15 with master changes right now .. will let you know when its done.

**Pull request merged**